src/error.rs

```diff
@@ -120,7 +120,7 @@ pub enum NotifyServerError {
     AccountNotAuthorized,
 
     #[error("sqlx error: {0}")]
-    SqlxError(#[from] sqlx::error::Error),
+    Sqlx(#[from] sqlx::error::Error),
```

Comment:
minor: would it be too costly to do for all error variants? i.e. are they referenced a lot?

Reply:
You're suggesting expand the `Sqlx` error here into all of the enum variants from `sqlx::error::Error`? Not sure why that would be useful

Reply:
Was referring to removing the Error suffix as as it's already implied by the enum's name. Should have been more explicit with my question.

Reply:
I'll do it with the other variants too at some point